jekyll/_cci2/creating-orbs.md

```diff
@@ -17,126 +17,62 @@ This guide covers the steps required to publish an orb.
 ## Introduction
 {: #introduction }
 
-After authoring your orb, you can publish it with a [semantically versioned]({{site.baseurl}}/2.0/orb-concepts/#semantic-versioning) tag, and the orb will appear publicly on the [Orb Registry](https://circleci.com/developer/orbs).
+After authoring your orb, you can publish it with a [semantically versioned]({{site.baseurl}}/2.0/orb-concepts/#semantic-versioning) tag, and the orb will appear on the [Orb Registry](https://circleci.com/developer/orbs).
+
+**Note:** If your orb is private, it will not be searchable on the Orb Registry, however the URL will be accessible to authenticated users of the orb.
```

Comment:
```suggestion
**Note:** If your orb is private, it will not be searchable on the Orb Registry. However, the URL will be accessible to authenticated users of the orb.
```